src/main/java/uk/gov/mca/beacons/api/controllers/OwnerController.java

```diff
@@ -41,4 +50,13 @@ public OwnerController(
 
     return beaconPersonMapper.toWrapperDTO(createOwnerService.execute(person));
   }
+
+  @GetMapping(value = "/{id}")
+  public WrapperDTO<BeaconPersonDTO> getOwner(@PathVariable("id") UUID id) {
+    final Person owner = getPersonByIdService.execute(id, PersonType.OWNER);
+
+    if (owner == null) throw new ResourceNotFoundException();
```

Comment:
As discussed, this will read better if you return an `Optional<Owner>` as you can do:

`!owner.isPresent() throw new ResourceNotFoundException();`